mcs/packages/Makefile

```diff
@@ -6,10 +6,10 @@ ROSLYN_CSC = Microsoft.Net.Compilers.1.2.0-rc/tools
 ifeq ($(PROFILE),basic)
 
 # Workaround for WARNING: Error: TrustFailure (The authentication or decryption has failed.) problem
```

Comment:
I just looked through the log in https://jenkins.mono-project.com/job/test-mono-pull-request-i386/2317/parsed_console/log_content.html#ERROR1 and it is using the system Mono, which is the current alpha 4.4.0.40 which has basically broken SSL on Linux. It was fixed with https://github.com/mono/mono/commit/04eb667e1bc4282a22f291b39099b23611793851 so this is expected until the next alpha hits :(